src/test/java/core/BaseTest.java

```diff
@@ -54,19 +55,19 @@ public void goToLandingPage() {
         landingPage = PageFactory.initElements(driver, LandingPage.class);
 
         if(landingPage.waitUntilExistXpath("//h1[contains(text(), 'Log in to your account')]")) {
+            dashboardPage = PageFactory.initElements(driver, DashboardPage.class);

```

Comment:
Please move declineCookies to other pages but don't instantiate DashboardPage before logging in.